ecomap/ecomap.xcodeproj/project.pbxproj

```diff
@@ -852,7 +852,7 @@
 					};
 					B35343341A7F73ED00DD2F03 = {
 						CreatedOnToolsVersion = 6.2;
-						DevelopmentTeam = FD442QNEZS;
+						DevelopmentTeam = 7VZHFC2543;
 						TestTargetID = B35343181A7F73ED00DD2F03;
```

Comment:
Generally, not really happy with the structure. You can take a look on any open source iOS project or on some samples from Apple, any framework, or some other. There should be groups like Views, View Controllers, Context, Models and so on, names is on your choose, but it should be structure that will be easy understandable. For now it's not.